ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -697,12 +698,13 @@ def test_frontend_function(
         else:
             # the function provides inplace update by default
             # check if returned reference is inputted reference
+            copy_kwargs["as_ivy_arrays"] = False
             first_array = ivy.func_wrapper._get_first_array(
                 *args, array_fn=array_fn, **kwargs
             )
             ret_ = get_frontend_ret(frontend_fn, *args, **kwargs)
             assert first_array is ret_
-            args, kwargs = copy_args, copy_kwargs
```

Comment:
I think this line existed so it `args` and `kwargs` gets reset to what they were before arguments were modified in place, is that not the case anymore?

Reply:
They appear to be unused 
![image](https://github.com/unifyai/ivy/assets/111577222/880ca9ca-d2fb-457d-9a91-cc29b56501cf)

Reply:
In that case, feel free to remove it, good work @AnnaTz !